src/views/Dashboard.vue

```diff
@@ -19,21 +19,15 @@
           >
             <!-- Alerts section-->
             <section
-              v-if="alertCount > 0"
-              id="dashboard-alerts-section"
-              class="mb-n6"
+              v-show="alertCount > 0"
+              id="alerts-section"
             >
               <header>
                 <h2 class="mb-3">
                   <span>Alerts</span>&nbsp;<span class="section-count">({{ alertCount }})</span>
                 </h2>
               </header>
-              <!-- FUTURE: move these to a new Alerts component and inside one expansion panel -->
-              <Amalgamation v-if="isAmalgamationAlert" />
-              <FrozenInformation v-if="isFrozenInformationAlert" />
-              <MissingInformation v-if="isMissingInformationAlert" />
-              <NotInCompliance v-if="isNotInComplianceAlert" />
-              <NotInGoodStanding v-if="isNotInGoodStandingAlert" />
+              <Alerts @count="alertCount = $event" />
```

Comment:
Awesome 💪

Reply:
Yes, this is a nice, concise API between the parent (Dashboard) and child (Alerts) components. This is simple enough that I didn't want to / need to store the alert count in the store.

Reply:
Yup, makes a lot of sense. Thanks Sev!